test/libsolidity/syntaxTests/literalSuffixes/parameters/exponent_out_of_range.sol

```diff
@@ -0,0 +1,31 @@
+function mi8eu8(int8 m, uint8 e) pure suffix returns (uint) {}
+function u8(uint m, uint8 e) pure suffix returns (uint) {}
+function u16(uint m, uint16 e) pure suffix returns (uint) {}
+function u256(uint m, uint e) pure suffix returns (uint) {}
+
+contract C {
+    function min() public pure {
+        // We never use positive exponents so here it's just 0 and the whole number goes into mantissa
+        1.0e78 u8;
+        1.0e78 u16;
+        1.0e78 u256;
+
+        115792089237316195423570985008687907853269984665640564039457584007913129639936 u8;   // 2**256
+        115792089237316195423570985008687907853269984665640564039457584007913129639936 u16;  // 2**256
+        115792089237316195423570985008687907853269984665640564039457584007913129639936 u256; // 2**256
+    }
+
+    function max() public pure {
+        1e-256 u8;
+        128e-256 mi8eu8;
```

Comment:
just wondering why here you don't use `1e-1234 u16;`

Reply:
I do, but `1234` is not out of range of a `uint16` exponent so that case is in `exponent_in_range.sol`.